src/docs/pam_krb5_migration.rst

```diff
@@ -0,0 +1,186 @@
+
+Migrating from pam_krb5
+=======================
+
+``pam_krb5`` was a Pluggable Authentication Module (PAM) for performing user
+session authentication against Kerberos (specifically krb5). Red Hat formally
+announced its deprecation in the RHEL-7.4 release.
+
+System Security Services Daemon (SSSD) is a broader toolsuite for managing
+authentication mechanisms and remote directories. It includes a PAM module,
+``pam_sss``, which can perform the tasks for which ``pam_krb5`` was previously
+used.
+
+Architecturally, ``pam_krb5`` was a monolithic module which performed all needed
+tasks within itself. SSSD is set up differently: the module ``pam_sss`` calls
+out to the running SSSD itself for most functionality.
+
+.. note::
+    SSSD does not currently handle the AFS capabilitqies of ``pam_krb5``. If you
+    would like such functionality, please contact Red Hat Support.
+
+Core functionality
+------------------
```

Comment:
Perhaps this section should be called ``Configuration`` instead?

Reply:
Agree.